public/index.html

```diff
@@ -4,10 +4,17 @@
   <meta charset="utf-8">
   <title>Steam Item Details Page</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500;700;900&display=swap" rel="stylesheet">
```

Comment:
I'm not sure if you're utilizing all 4 specified font weights, but I believe you can reduce the size of the asset download when only specifying weights that are being used.

[I saw a font-weight declaration here](https://github.com/FEC-Bell/christina-proxy/pull/2/files#diff-8a86461355201e53996350d19f6ef43fR45-R53) for 300. Not sure if the other weights are being used.

Reply:
Will remove unneeded font weights in my next PR! After near-future deliverable deadlines, I'm also planning on looking into code splitting to further reduce production file sizes.